### DISCUSSION TITLE
phpRedisAdmin stats please

### DISCUSSION THREAD

**Discussion by GerardEarley** [erikdubbelboer/phpRedisAdmin#208] [Ideas]
Dear Erik,
phpRedisAdmin  is extremely useful to look at a redis instance's keys, it's super useful for that. But I can't help but feel that that is of limited use to most people.

Having the relative health of various metrics presented *visually* would be super useful.

For instance I have redis caching the backend of a number of wordpress websites but I've no idea how well I've set the cache size. I know I can go into the docs and look up all the stats that redis provides but I cant help but feel phpRedisAdmin  should be showing that on the front page. Just like opcache-gui does for php

https://duckduckgo.com/?q=php+opcache-gui+&t=ffab&iar=images&iax=images&ia=images

Please do consider it.

**Comment by erikdubbelboer** [comment]
I'm afraid I don't have time to add something like that, but a pull request is always welcome!

**Comment by tessus** [comment]
Have a look at https://github.com/tessus/valkey-stats

I wrote that a while back. It doesn't require a php redis module and communicates directly via the socket (unix/ip), thus has some limitations. However, one can easily get all that info via the redis module, adjust the code accordingly, and add it to phpRedisAdmin. In fact, I've been wanting to use the module (if loaded by PHP, otherwise ues the current way), but I never got around to add it.
